scripts/semconv/generate.sh

```diff
@@ -4,39 +4,44 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOT_DIR="${SCRIPT_DIR}/../../"
 
 # freeze the spec version to make SemanticAttributes generation reproducible
-SPEC_VERSION=v1.11.0
-OTEL_SEMCONV_GEN_IMG_VERSION=0.11.1
+SPEC_VERSION=v1.21.0
+SCHEMA_URL=https://opentelemetry.io/schemas/$SPEC_VERSION
+OTEL_SEMCONV_GEN_IMG_VERSION=0.21.0
 
 cd ${SCRIPT_DIR}
 
-rm -rf opentelemetry-specification || true
-mkdir opentelemetry-specification
-cd opentelemetry-specification
+rm -rf semantic-conventions || true
+mkdir semantic-conventions
+cd semantic-conventions
 
 git init
-git remote add origin https://github.com/open-telemetry/opentelemetry-specification.git
+git remote add origin https://github.com/open-telemetry/semantic-conventions.git
 git fetch origin "$SPEC_VERSION"
 git reset --hard FETCH_HEAD
 cd ${SCRIPT_DIR}
 
 docker run --rm \
-  -v ${SCRIPT_DIR}/opentelemetry-specification/semantic_conventions/trace:/source \
+  -v ${SCRIPT_DIR}/semantic-conventions/model:/source \
   -v ${SCRIPT_DIR}/templates:/templates \
   -v ${ROOT_DIR}/opentelemetry-semantic-conventions/src/opentelemetry/semconv/trace/:/output \
   otel/semconvgen:$OTEL_SEMCONV_GEN_IMG_VERSION \
+  --only span,event,attribute_group \
```

Comment:
Not too familiar with docker syntax. Does the `--only` flag filter by type? How does it know to match by "type" field?

Reply:
that's build-tools (semconvgen) command line syntax.
`--only` flag allows filtering types of semantic conventions to generate in the output - here's the list of all supported conventions - https://github.com/open-telemetry/build-tools/blob/main/semantic-conventions/syntax.md#syntax (under `convtype`)